### ISSUE TITLE
3D Tiles 1.0 Updates

### ISSUE THREAD

**Issue by lilleyse** [AnalyticalGraphicsInc/cesium#6697]
These are the updates required to load tilesets defined by the [3D Tiles 1.0 spec](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/1.0).
@ggetz if you think of others feel free to add to this list.

**Now**:
* [x] Add support for b3dm RTC_CENTER #6488. The glTF 1.0 extension `CESIUM_RTC` will still be supported but this is the preferred method.
* [x] content.url changed to content.uri. Support loading both, but if content.url is used log a deprecation warning. Once the PR is merged, add a note to #5363.
* [x] Load batch table hierarchy extension. Still support loading the non-extension version, but log a deprecation warning. Once the PR is merged, add a note to #5363.
* [x] Update all tilesets in `Specs/Data/Cesium3DTiles` with the upgraded tilesets in 3d-tiles-tools: #5504 and https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/pull/139

**Later**:
* [ ] Add extras to API #6490. No need to add extensions to API yet.
* [ ] Update any 3D Tiles links in the documentation after the [1.0](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/1.0) branch is merged.
* [ ] Standardize the deprecation messages for anything in https://github.com/AnalyticalGraphicsInc/cesium/issues/5363.

**Comment by lilleyse** [comment]
In the near future we should also go through all the items in https://github.com/AnalyticalGraphicsInc/cesium/issues/5363 and remove the deprecation checks.

**Comment by ggetz** [comment]
@lilleyse I double checked and this list seemed comprehensive. The only thing else I can think of is if the padding clarifications require any implementation changes, which I don't believe they do.

**Comment by hpinkos** [comment]
@lilleyse all of our tilesets used in Sandcastle generate deprecation warnings.  Is fixing that part of this roadmap issue?

**Comment by ggetz** [comment]
@hpinkos https://github.com/AnalyticalGraphicsInc/cesium/issues/5504

3d-tiles-tools has been updated to generate 1.0 tilesets, so we should be good to at least get rid of the sandcastle tilesets that throw deprecation warnings before the next release.

**Comment by mramato** [comment]
Just a reminder that the release is this Wednesday.  We should really try and get all showstopper issues addressed by the end of today.

**Comment by tfili** [comment]
Is this going to make it in for tomorrow?

**Comment by ggetz** [comment]
See https://github.com/AnalyticalGraphicsInc/cesium/pull/6862 and https://github.com/AnalyticalGraphicsInc/cesium/pull/6861#issuecomment-409052452

I think that covers all sandcastle tilesets.

**Comment by tfili** [comment]
What about the extra tasks under later? Do you want to write a new issue or just remove the `Next Release` tag on this one?

**Comment by mramato** [comment]
We really should be writing small issues, no roadmaps.  Roadmaps make everything super opaque and hard to fine.

**Comment by lilleyse** [comment]
Ok this is good to close now.

* `extras` API - #6490 is still open
* Update 3D Tiles links - part of https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/327
* Deprecation messages https://github.com/AnalyticalGraphicsInc/cesium/issues/5363 - closed

**Comment by cesium-concierge** [comment]
Congratulations on closing the issue! I found these Cesium forum links in the comments above:

https://groups.google.com/forum/#!topic/cesium-dev/8zJhev5Vh5w

If this issue affects any of these threads, please post a comment like the following:

> The issue at https://github.com/AnalyticalGraphicsInc/cesium/issues/6697 has just been closed and may resolve your issue. Look for the change in the next stable release of Cesium or get it now in the master branch on GitHub https://github.com/AnalyticalGraphicsInc/cesium.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:
